src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -38,6 +40,7 @@ class IndexManagerTests extends HyperspaceSuite with SQLHelper {
 
   override def beforeAll(): Unit = {
     super.beforeAll()
+    spark.conf.set(HYPERSPACE_EVENT_LOGGER_CLASS_KEY, "com.microsoft.hyperspace.MockEventLogger")
```

Comment:
for reviewers: setting this config at the beginning of tests because it's a `lazy val` in a `object EventLogger` class, and hence it is initialized only once at the beginning of the test suite.

I tried setting this within a test but since the lazy val got initialized before this config could take effect, it did not work.